### ISSUE TITLE
Why imagequant.dll require MessageBoxW function? 

### ISSUE THREAD

**Issue by zedxxx** [jibsen/pngquant-winbuild#4]
Why `imagequant.dll` (at least 32-bit version) require `MessageBoxW` function from `user32.dll`?

![image 1](https://cloud.githubusercontent.com/assets/6298130/18609243/1fe71046-7d06-11e6-8248-23dede51cfaa.png)

**Comment by jibsen** [comment]
It looks like the place in the dll that calls `MessageBoxW` uses the (unicode) string "MinGW Runtime Assertion". Googling that suggests it comes from an assertion in the [MinGW crt runtime](https://sourceforge.net/p/mingw-w64/code/HEAD/tree/trunk/mingw-w64-crt/misc/wassert.c).

**Comment by zedxxx** [comment]
I found that if I compile lib without `openmp` support, then it don't link `user32.dll` and resulted dll size is 3 times smaller (~70 kB, instead of 220 kB). But, of course, this version will be slower.

I attracted attention on this, because I have old version of  `imagequant` compiled couple of years ago with old MinGW (not MinGW-w64) and this dll don't require `user32.dll`. I'm not sure, about `openmp` support in this dll, but it size is about 200 kB too.
